### DISCUSSION TITLE
db file<title>

### DISCUSSION THREAD

**Discussion by littlemoon0909** [community/community#45384] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello everyone,

I'm new to SQL. I downloaded a .db file from GitHub but unable to import it into my Postgresql pgAmdin 4 platform. It keeps saying 'invalid archive file'. Can I create a database in Postgresql using an existing .db file or do i have to convert it to something else? I would appreciate any help!

Thank you!

**Comment by jge162** [comment]
use "pgloader" to convert and import the data directly from SQLite to PostgreSQL.

**Reply by littlemoon0909** [reply]
Thank you so much for your reply! It seems like pgloader is the answer to my question. However, I was unable to install pgloader. I downloaded a couple of different packages, but some have no 'install' file or the 'install' file didn't work. I also used the terminal to install but didn't work. I looked for instructions on those websites where I downloaded the packages, but they were all so confusing. I'm new to coding and stuffs, would you please give me a step by step instruction on how to install pgloader?  Thank you!

**Reply by jge162** [reply]
First, make sure you have PostgreSQL installed on your system. If not, you can download and install it from the official website 

(https://www.postgresql.org/download/).

Next, download the pgloader package from the official website 

(https://github.com/dimitri/pgloader). 
You can either download the source code or a pre-built binary package.

If you downloaded the source code, extract the archive and navigate to the extracted directory in your terminal.

Run the following command to install pgloader->

$ make install

If you downloaded a pre-built binary package, extract the archive and navigate to the extracted directory in your terminal.

Run the following command to install pgloader->

$ sudo make install

To confirm that pgloader has been installed successfully, run the following command->

$ pgloader --version

You should see the version number of pgloader displayed in the terminal.

If you have any other GitHub related questions please lmk. Thanks.
